### ISSUE TITLE
examples/Meteorites.csv missing

### ISSUE THREAD

**Issue by jtelszasz** [pandas-profiling/pandas-profiling#155]
Tried running the meteorites.ipynb example but it seems the Meteorite_Landings.csv has gone missing!

The second cell in the meteorites.ipynb would load it:
`df=pd.read_csv("examples/Meteorite_Landings.csv", parse_dates=['year'], encoding='UTF-8')`

But here's the error:
`FileNotFoundError: File b'examples/Meteorite_Landings.csv' does not exist`

(Or, you could just look in the directory and see that it isn't there. :) )

**Comment by igorbrigadir** [comment]
Since the data the notebook links to is 5MB https://data.nasa.gov/api/views/gh4g-9sfh/rows.csv?accessType=DOWNLOAD i assume it wasn't included because it would make the repository unnecessarily large. The notebook does link to the data page, but not directly to the CSV.

In newer versions of pandas (`>0.19.2`) you can open directly from web: 

```
df = pd.read_csv("https://data.nasa.gov/api/views/gh4g-9sfh/rows.csv?accessType=DOWNLOAD", parse_dates=['year'], encoding='UTF-8')
```

**Comment by JosPolfliet** [comment]
Great suggestion, implemented in ade6567207b7df52b9ee220d8e5f2944d91d9acc
